install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml

```diff
@@ -0,0 +1,37 @@
+MultiBeamSEM: {
+    class: Microscope,
+    role: mbsem,
+    children: ["ASM", "EBeamScanner", "MirrorDescanner", "MPPC",],
+}
+
+"ASM": {
+  class: technolution.AcquisitionServer,
+  role: "main",
+  init: {
+    # Use following host url for real system instead of simulator: "http://192.168.10.10:8080/v2"
+    host: "http://localhost:8080/v2",
```

Comment:
Do we want as default the simulator or the real system url?

Reply:
For such file, it'd be the real system IP.

Reply:
I agree it should be the real system IP as this yaml file is placed in the hwtest folder. All simulator yaml files are in the sim folder.